fluid-music/src/groove.js

```diff
@@ -30,27 +29,27 @@ const getGroove = function(groovelib, grooveName){
 const applyGroove = function(notes, grooveItem, multipliers, randomness=0) {
     if (typeof grooveItem === "string"){
         selG = getGroove(grooves, grooveItem);
-        if (selG === null) throw new Error ("Groove Not Found")
+        if (selG === null) throw new Error ("Groove Not Found");
     }   
     else if (typeof grooveItem === "object")
         selG = grooveItem;
     else
         throw new Error("grooveItem must either be a string or valid groove object");
 
     if (typeof randomness !== "number")
-        throw Error("randomness must be a number.")
+        throw Error("randomness must be a number.");
     if (typeof multipliers === "object"){
         multipliers.v = multipliers.v ? multipliers.v : 1;
         multipliers.o = multipliers.o ? multipliers.o : 1;
-        multipliers.level = multipliers.level ? multipliers.level : 1;
+        multipliers.l = multipliers.l ? multipliers.l : 1;
```

Comment:
Remember that `multipliers.o ? : multipliers.o  : 1` will evaluate to `1` if `multipliers.o === undefined` OR `multipliers.o === 0` . 0 multiplier but be undesirable, so that might be correct. @zhiweigan, can you verify that this is the desired behavior.

Reply:
Sometimes in this case you really want to something like 

```
multipliers.o = typeof multipliers.o === 'number' ? multipliers.o : 1;
```

... depending on the desired behavior.

Reply:
That was an oversight on my part, I have just committed another change to the request.